build/configs/rtl8721csm/hello/Make.defs

```diff
@@ -210,6 +210,13 @@ define MAKE_BOARD_SPECIFIC_BIN
   $(TOPDIR)/../build/configs/rtl8721csm/rtl8721csm_make_bin.sh
 endef
 
+SECURE_HEADER_SIZE=20
+
+define MAKE_SAMSUNG_HEADER
+  $(TOPDIR)/../os/tools/mkbinheader.py $(TOPDIR)/../build/output/bin/km0_km4_image2.bin kernel ${SECURE_HEADER_SIZE}
```

Comment:
I suggest two things.
1. Let's make new script file to call these two lines. 
   And please write entire structure of samsung header there. There is no total structure of samsung header anywhere.
2. Let's make these two lines can be used in other board. It means you should use some macro for binary like `SECURE_HEADER_SIZE`.
  It's easier to know what we should modify for new board.